Corona-Warn-App/src/androidTest/java/testhelpers/ScreenShotter.kt

```diff
@@ -29,19 +26,11 @@ inline fun <reified T> takeScreenshot(suffix: String = "", delay: Long = SCREENS
     Thread.sleep(delay)
     val simpleName = T::class.simpleName
     val name = if (suffix.isEmpty()) simpleName else simpleName.plus("_$suffix")
-
-    val contentResolver = getInstrumentation().targetContext.contentResolver
-    val testLabSetting = Settings.System.getString(contentResolver, "firebase.test.lab")
-    val androidStudioMode = InstrumentationRegistry.getArguments().getString("androidStudioMode")
-    if ("true" in listOf(testLabSetting, androidStudioMode)) {
-        Screengrab.screenshot(
-            name,
-            UiAutomatorScreenshotStrategy(),
-            SDCardCallback
-        )
-    } else {
-        Screengrab.screenshot(name)
-    }
+    Screengrab.screenshot(
```

Comment:
Can you please update our Wiki now that we don't need to set `androidStudioMode` anymore?